plugins/modules/system/launchd.py

```diff
@@ -0,0 +1,484 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Martin Migasiewicz <migasiew.nk@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: launchd
+author:
+- Martin Migasiewicz (@martinm82)
+version_added: "0.1.1"
```

Comment:
```suggestion
version_added: "0.1.1"
```
@felixfontein  ^ is the version ok?

Reply:
I removed this for now since there is no decision on versioning schema.